scripts/testimonials.js

```diff
@@ -1,62 +1,36 @@
-const mySiema = new Siema({
-  selector: '.carousel--items',
-  duration: 200,
-  easing: 'ease-out',
-  perPage: 1,
-  startIndex: 0,
-  draggable: true,
-  multipleDrag: true,
-  threshold: 20,
-  loop: false,
-  rtl: false,
-  onInit: () => {},
-  onChange: () => {},
-})
+function setUpCarousel(root) {
+    const siema = new SiemaWithDots({
+        selector: root.querySelector('.carousel--items'),
+        duration: 200,
+        easing: 'ease-out',
+        perPage: 1,
+        startIndex: 0,
+        draggable: true,
+        multipleDrag: true,
+        threshold: 20,
+        loop: true,
+        onInit: function() {
+            this.addDots()
```

Comment:
having the dot adding code within siema makes it possible to refer to the siema object as `this`, and reference it within `onInit` and `onChange` (which in turn saves us from having to set up our own click handlers)